api/attribute.proto

```diff
@@ -391,7 +391,7 @@ message MUPType1SessionTransformedRoute {
   // - RouteDistinguisherIPAddress
   // - RouteDistinguisherFourOctetASN
   google.protobuf.Any rd = 1;
-  uint32 prefix_length = 2;
+  uint32 prefix_length = 2 [deprecated = true];
```

Comment:
No code uses this field?

Reply:
Yes.

Reply:
There are 3 options to remove existing field.

1. Remove the field.
2. [Add deprecated field option](https://developers.google.com/protocol-buffers/docs/proto3#options).
3. [Reserve the field number](https://developers.google.com/protocol-buffers/docs/proto3#reserved).

Which one is better for the gobgp API?